wiki/markdown/images/multiple_EI_instances.svg

```diff
@@ -571,7 +571,7 @@
        x="643.23022"
        y="294.7197"
        style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:9.44525242px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:center;writing-mode:lr-tb;text-anchor:middle;stroke-width:0.55391979"
-       id="tspan6561-8">aggregated.collection.name: rule1</tspan></text>
+       id="tspan6561-8"> aggregations.collection.name: rule1</tspan></text>
```

Comment:
I assume only the svg files have been updated? The png files should also be I guess.

Reply:
Yes but I didn't bother with that yet since I wanted to wait a bit for feedback.

Reply:
Done